<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetForm</class>
 <widget class="QWidget" name="SetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="SetTabWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>39</y>
     <width>711</width>
     <height>451</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QGroupBox" name="IPgroupBox">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>601</width>
       <height>179</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="AutoSetIPradioButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>DHCP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ManualSetIPradioButton">
          <property name="text">
           <string>Static</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="IPlabel">
            <property name="text">
             <string>网络地址</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="IPlineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="ChildIPlabel">
            <property name="text">
             <string>子网掩码</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ChildIPlineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="GetWaylabel">
            <property name="text">
             <string>网关地址</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="GetWaylineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="DNSgroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>601</width>
       <height>179</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="AutoSetDNSradioButton">
          <property name="text">
           <string>DHCP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ManualSetDNSradioButton">
          <property name="text">
           <string>Static</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="DNS1label">
            <property name="text">
             <string>DNS1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="DNS1lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="DNS2label">
            <property name="text">
             <string>DNS2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="DNS2lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="DNS3label">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>DNS3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="DNS3lineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="ServerIPlabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>服务器IP</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="ServerIPlineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="RoomNumlabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>84</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="SeatNumlabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>137</y>
       <width>84</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="RoomNumcomboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="SeatlineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>131</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Tiplabel22">
     <property name="geometry">
      <rect>
       <x>118</x>
       <y>250</y>
       <width>421</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="title_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>697</x>
      <y>9</y>
      <width>16</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image1/x.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>497</y>
     <width>711</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="SavepushButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="CancelpushButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
